hub/parameters/digitalhubprd.parameters.json

```diff
@@ -0,0 +1,15 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "databaseAccounts_digitalhub_name": {
+            "value": "digitalhubprd"
```

Comment:
The beauty of this is that we can simply create a new paramters file and it will build it with whatever name we want.